src/apps/bm_soft_module/user_code/user_code.cpp

```diff
@@ -41,49 +46,59 @@ static bool BmSoftStartAndValidate(void);
 static int createBmSoftDataTopic(void);
 static void BmSoftInitalize(void);
 
-void setup(void) {
-  configASSERT(userConfigurationPartition);
-  configASSERT(sysConfigurationPartition);
-  BmSoftInitalize();
+static void getConfigs() {
+  if (sysConfigurationPartition->getConfig(soft_cfg_tsys_id, strlen(soft_cfg_tsys_id),
+                                           serial_number)) {
+    printf("TSYS serial number: %" PRIu32 "\n", serial_number);
+  } else {
+    printf("No TSYS serial number found\n");
+    bm_fprintf(0, soft_log, "No TSYS serial number found\n");
+    bm_printf(0, "No TSYS serial number found");
+  }
+
   int32_t calTempC = 0;
-  int32_t calTempMilliC = 0;
-  if (sysConfigurationPartition->getConfig("calTempC", strlen("calTempC"), calTempC)) {
-    printf("calTempC: %" PRId32 "\n", calTempC);
+  if (sysConfigurationPartition->getConfig(soft_cfg_cal_temp_c, strlen(soft_cfg_cal_temp_c),
```

Comment:
This runs contrary to the product spec, but if this is an informational term, this should be an `RO-app-write` and thus a `configSet` post computation of the milliDegC and reflect the system rather than a `R/W` by the calibrator.

Reply:
@russelldeguzman @towynlin - I'm having trouble following the thread here.
This just seems like a debug print of the calTemp value. Can you clarify the conflict with the product spec?

Reply:
@evanShap The table shows R/W mutability on the spec at the moment - is the intention of this value to be set by the app? or be written at calibration time?

Reply:
I confirmed with Evan that this is set by the production tooling when calibration is done. It's just information about our calibration process.

Reply:
@towynlin @russelldeguzman Yep - I added more description to the Notes column for all of these.

Additional note – the R/W mutability column is a future-facing spec. I didn't expect we have a clean way of enforcing these currently, and that's not required. So today R/W describes intended behavior, but not an expectation we'll actually prevent writes via `bm cfg` commands to RO configs.